src/test/java/io/odh/test/e2e/standard/MultipartFormDataBodyPublisherTest.java

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright Skodjob authors.
+ * Copyright (c) 2022 yusuke suzuki
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.odh.test.e2e.standard;
+
+import com.sun.net.httpserver.HttpServer;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse.BodyHandlers;
+import java.nio.ByteBuffer;
+import java.nio.channels.Channels;
+import java.nio.channels.ReadableByteChannel;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.security.DigestInputStream;
+import java.security.MessageDigest;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class MultipartFormDataBodyPublisherTest {
```

Comment:
it is a unit test? if so it should be in unit test package

Reply:
can't put it there

```
Method MultipartFormDataBodyPublisherTest.testMultipartFormDataChannel() uses package-private interface io.odh.test.platform.httpClient.Part  Method MultipartFormDataBodyPublisherTest.testMultipartFormDataChannelException() uses package-private interface io.odh.test.platform.httpClient.Part  Method MultipartFormDataBodyPublisherTest.testStreamPart() uses package-private class io.odh.test.platform.httpClient.StreamPart  Method MultipartFormDataBodyPublisherTest.testMultipartFormDataChannelException() uses package-private class io.odh.test.platform.httpClient.StreamPart  Method MultipartFormDataBodyPublisherTest.testFilePart() uses package-private class io.odh.test.platform.httpClient.FilePart  Method MultipartFormDataBodyPublisherTest.testStringPart() uses package-private class io.odh.test.platform.httpClient.StringPart  Method MultipartFormDataBodyPublisherTest.testMultipartFormDataChannel() uses package-private class io.odh.test.platform.httpClient.StringPart  
```

I cant put it under `unit`, it has to be in `package io.odh.test.platform.httpClient` otherwise it can't test these package private things.

Reply:
do you need to test private methods?

Reply:
package-private, yes; at least, the japanese guy who wrote it wrote it this way

Reply:
udk what to say. I won't accept and merge it until you move it to unit package.